test/testcommon/src/main/java/io/pravega/test/common/JwtTestUtils.java

```diff
@@ -7,20 +7,24 @@
  *
  *     http://www.apache.org/licenses/LICENSE-2.0
  */
-package io.pravega.client.security.auth;
+package io.pravega.test.common;
 
 import com.google.gson.Gson;
 
 import java.util.Base64;
 
 public class JwtTestUtils {
 
-    static String createJwtBody(JwtBody jwt) {
+    public static String createJwtBody(JwtBody jwt) {
```

Comment:
Javadoc for this class and all its methods.

Reply:
Done.